README.md

```diff
@@ -134,7 +134,14 @@ An array of deep linking urls and/or errors in case invalid params.
     
 ## Structuring a 'dynamic' Deeplink
 
-This should be used for situations where the longer link is okay and you want to create links quickly without a POST to the API.
+This should be used for situations where the longer link is okay and you want to create links quickly without a POST to the API. Here's a list of instructions on how to build a deep link: 
+
+- Start with your Branch domain, http://bnc.lt (or your white labeled one). 
+- Append /a/your_Branch_key.
+- Append the start of query params '?' 
+- Append the Branch analytics tag feature=marketing&channel=email&tags[=drip1&tags[]=welcome 
```

Comment:
Okay, thanks--will do.

On Fri, Jun 5, 2015 at 5:05 PM, Alex Austin <notifications@github.com>
wrote:

> In README.md
> <https://github.com/BranchMetrics/Branch-Public-API/pull/8#discussion_r31861079>
> :
>
> > @@ -134,7 +134,14 @@ An array of deep linking urls and/or errors in case invalid params.
> >
> >  ## Structuring a 'dynamic' Deeplink
> >
> > -This should be used for situations where the longer link is okay and you want to create links quickly without a POST to the API.
> > +This should be used for situations where the longer link is okay and you want to create links quickly without a POST to the API. Here's a list of instructions on how to build a deep link:
> > +
> > +- Start with your Branch domain, http://bnc.lt (or your white labeled one).
> > +- Append /a/your_Branch_key.
> > +- Append the start of query params '?'
> > +- Append the Branch analytics tag feature=marketing&channel=email&tags[=drip1&tags[]=welcome
>
> Let's add one line here:
> Append the data parameter (base64 encoded) filled with your Branch control
> parameters - see table of them here (link to above table)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/BranchMetrics/Branch-Public-API/pull/8/files#r31861079>
> .
>